### ISSUE TITLE
Remove or consolidate unused column tags 

### ISSUE THREAD

**Issue by LukeSchlangen** [abamath/abamath.com#21]
I have a lot of extra tags from the post elements when I transitioned over from blogger. I would love to have these removed or consolidated. I would imagine that all of this:

```
<body id='page-top'>
<div class='navbar navbar-default navbar-fixed-top' id='top'>
<div class='section' id='section1'>
<div class='widget Image' data-version='1' id='Image2'>
<div class='widget-content'>
<a href='http://www.abamath.com/'>
<img alt='Logo Image in Top Left' id='Image2_img' src='public/img/abamath-logo.png'/>
</a>
<br/>
</div>
</div>
</div>
<div class='section' id='pagescroll'>
<div class='widget Blog' data-version='1' id='Blog2'>
<button class='navbar-toggle' data-target='.navScrollHeaderCollapse' data-toggle='collapse'>
<span class='icon-bar'></span>
<span class='icon-bar'></span>
<span class='icon-bar'></span>
</button>
<div class='collapse navbar-collapse navScrollHeaderCollapse'>
<ul class='nav navbar-nav navbar-right'>
<li class='page-scroll'>
<a href='#classes'>classes</a>
</li>
<li class='page-scroll'>
<a href='#about'>about</a>
</li>
<li class='page-scroll'>
<a href='#contact'>contact</a>
</li>
</ul>
</div>
<div class='clear'></div>
</div>
</div>
</div>
<div class='content'>
<header>
<div class='header-outer'>
<div class='fauxborder-left header-fauxborder-left'>
<div class='fauxborder-right header-fauxborder-right'></div>
<div class='region-inner header-inner'>
<div class='header section' id='header'>
<div class='widget Header' data-version='1' id='Header1'>
<div id='header-inner' style='background-image: url("public/img/website-creation-for-kids.jpg"); background-position: center; background-repeat: no-repeat; background-size: cover; height: 320px; padding: 0px; margin: 0px; '>
</div>
<div class='clear'></div>
</div>
</div>
</div>
</div>
</div>
</header>
<div class='content-outer'>
<div class='content-cap-top cap-top'>
<div class='cap-left'></div>
<div class='cap-right'></div>
</div>
<div class='fauxborder-left content-fauxborder-left'>
<div class='fauxborder-right content-fauxborder-right'></div>
<div class='content-inner'>
<div class='main-outer'>
<div class='main-cap-top cap-top'>
<div class='cap-left'></div>
<div class='cap-right'></div>
</div>
<div class='fauxborder-left main-fauxborder-left'>
<div class='fauxborder-right main-fauxborder-right'></div>
<div class='region-inner main-inner'>
<div class='columns fauxcolumns'>
<div class='fauxcolumn-outer fauxcolumn-center-outer'>
<div class='cap-top'>
<div class='cap-left'></div>
<div class='cap-right'></div>
</div>
<div class='fauxborder-left'>
<div class='fauxborder-right'></div>
<div class='fauxcolumn-inner'>
</div>
</div>
<div class='cap-bottom'>
<div class='cap-left'></div>
<div class='cap-right'></div>
</div>
</div>
<div class='fauxcolumn-outer fauxcolumn-left-outer'>
<div class='cap-top'>
<div class='cap-left'></div>
<div class='cap-right'></div>
</div>
<div class='fauxborder-left'>
<div class='fauxborder-right'></div>
<div class='fauxcolumn-inner'>
</div>
</div>
<div class='cap-bottom'>
<div class='cap-left'></div>
<div class='cap-right'></div>
</div>
</div>
<div class='fauxcolumn-outer fauxcolumn-right-outer'>
<div class='cap-top'>
<div class='cap-left'></div>
<div class='cap-right'></div>
</div>
<div class='fauxborder-left'>
<div class='fauxborder-right'></div>
<div class='fauxcolumn-inner'>
</div>
</div>
<div class='cap-bottom'>
<div class='cap-left'></div>
<div class='cap-right'></div>
</div>
</div>
<!-- corrects IE6 width calculation -->
<div class='columns-inner container'>
<div class='column-center-outer'>
<div class='column-center-inner'>
<div class='main section' id='main'>
<div class='widget Blog' data-version='1' id='Blog1'>
<div class='blog-posts hfeed'>
<!--Can't find substitution for tag [defaultAdStart]-->

```

could be reduced to just a few lines since I don’t care about the meta data. This also shows up in multiple locations. I’m just scared to remove it quickly because I’m not sure if there will be styling side effects and still allowing my links to work (scrolling by clicking on the top nav).

**Comment by pbarnum** [comment]
I'll make an attempt for this one.

**Comment by ericorruption** [comment]
Is this complete already?

**Comment by LukeSchlangen** [comment]
@ericorruption

@pbarnum cleared up a lot of it, but there are still a few tags that can probably be removed cleanly to make the code simpler:

```HTML
    <div class='content-outer'>
      <div class='fauxborder-left content-fauxborder-left'>
        <div class='columns fauxcolumns'>
          <div class='columns-inner container'>
            <div class='main section' id='main'>
              <div class='widget Blog' data-version='1' id='Blog1'>

                <!--Can't find substitution for tag [defaultAdStart]-->
                <div class='post-outer' id='classes'>
                  <div class='post hentry'>
                    <div class='post-body entry-content'>
                      <!-- Upcoming Classes Section-->
                      <section class="content-section full-screen" id="classes">
```

**Comment by LukeSchlangen** [comment]
Fixed by #35
